src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -37,15 +38,34 @@
  * @author George Politis
  */
 public class BitrateController
+    extends PeriodicRunnable
     implements TransformEngine
 {
     /**
+     * The {@link Logger} to be used by this instance to print debug
+     * information.
+     */
+    private final Logger logger = Logger.getLogger(BitrateController.class);
+    
+    /**
+     * the interval/period in milliseconds at which {@link #run()} is to be
+     * invoked.
+     */
+    private static final long PADDING_PERIODICITY_MS = 15;
```

Comment:
this language also seems different from `PERIOD_MS` in https://github.com/jitsi/jitsi-videobridge/pull/418, but seems like they are similar and could be more consistent?